src/tests/ftest/vmd/led.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import json
+import time
+from osa_utils import OSAUtils
+from avocado.utils import process
+
+class VMDLEDStatus(OSAUtils):
+    """
+    Test Class Description: This class methods to get
+    the VMD LED status.
+    """
+    # pylint: disable=too-many-instance-attributes,too-many-ancestors
+    def setUp(self):
+        super().setUp()
+        self.targets = self.params.get("targets", "/run/server_config/servers/0/*")
+        self.dmg = self.get_dmg_command()
+        self.dmg.hostlist = self.hostlist_servers[0]
+
+    def get_nvme_device_ids(self):
+        """Get the list of nvme device-ids.
+        Returns: List of uuid
+        """
+        self.dmg.json.value = True
+        try:
+            result = self.dmg.storage_query_list_devices()
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
```

Comment:
If the storage_query_list_devices() method uses self._get_json_result() then these commands will not be required as they are handled by the method - including storing the original value and returning it to its previous value:
```suggestion
        try:
            data = self.dmg.storage_query_list_devices()
        except process.CmdError as details:
            self.fail("dmg command failed: {}".format(details))
```

Reply:
These are ujsed on other test cases. Going to leave the output as self._get_result for now.